Toolset/palettes/tutorial/courses/First Run/tutorials/Combining Text/lessons/App Lesson 0.txt

```diff
@@ -0,0 +1,377 @@
+tutorial "Combining Text"
+
+prologue
+	LiveCode's uniquely powerful text handling capabilities comes in handy for a wide range of applications. Common tasks include:
+
+	- Mail merging a form email
+
+	- Creating a leaflet for print
+
+	- Building an invoice
+end prologue
+
+step "Create Mainstack"
+
+action
+    highlight menu item "Default size" of menu item "New Stack" of menu "File"
+    capture the next new stack as "Mainstack"
+    capture the next new card of stack "Mainstack" as "home"
+    wait until there is a stack "Mainstack"
+		go to step "Set Stack Properties"
+end step
+
+step "Set Stack Properties"
+	Click on the Inspector icon on the toolbar to open the property
+	inspector.
+action
+	highlight toolbar "Inspector"
+	wait until there is an inspector for stack "Mainstack"
+	go to step "Set Stack Title"
+end step
+
+step "Set Stack Title"
+	Set the 'Title' of the stack to "Combine Text".
+action
+	highlight property "title" of section "basic"
+	wait until the title of stack "Mainstack" is "Combine Text"
+	go to step "Set Stack Rect"
+end step
+
+step "Set Stack Rect"
+	Resize the stack so that it has width 700 and height 500.
+action
+	add guide "Stack" with rect "0,0,700,500" to stack "Mainstack"
+	highlight guide "Stack"
+	wait until stack "Mainstack" fits guide "Stack" with tolerance 5
+	go to step "Set Stack Color"
+end step
+
+step "Set Stack Color"
+	Resize the stack so that it has width 700 and height 500.
```

Comment:
I know this isn't actually going to be run, but probably worth correcting the instruction text here